components/rtttl.rst

```diff
@@ -36,15 +37,40 @@ The tone generator needs a PWM capable output to work with, currently only the
     rtttl:
       output: rtttl_out
 
+Overview Using the I2S speaker
+------------------------------
+
+Using the tone generator with the :ref:`I2S Speaker <i2s_audio>` the following code sample is needed.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    i2s_audio:
+      - i2s_lrclk_pin: GPIO25
+        i2s_bclk_pin: GPIO5
+
+    speaker:
+      - platform: i2s_audio
+        id: board_external_speakers
+        dac_type: external
+        i2s_dout_pin: GPIO26
+        mode: mono
```

Comment:
I would rather not include the config for the speaker etc here to allow users to go and read the relevant pages about these. You could also remove the `output` from the above yaml too =)